apps/server/src/modules/video-conference/service/video-conference.service.ts

```diff
@@ -91,13 +110,17 @@ export class VideoConferenceService {
 	private async loadScopeRessources(
 		scopeId: EntityId,
 		scope: VideoConferenceScope
-	): Promise<Course | TeamEntity | null> {
-		let scopeRessource: Course | TeamEntity | null = null;
+	): Promise<ColumnBoard | Course | Room | TeamEntity | null> {
+		let scopeRessource: ColumnBoard | Course | Room | TeamEntity | null = null;
 
 		if (scope === VideoConferenceScope.COURSE) {
 			scopeRessource = await this.courseService.findById(scopeId);
 		} else if (scope === VideoConferenceScope.EVENT) {
 			scopeRessource = await this.teamsRepo.findById(scopeId);
+		} else if (scope === VideoConferenceScope.ROOM) {
+			scopeRessource = await this.roomService.getSingleRoom(scopeId);
+		} else if (scope === VideoConferenceScope.VIDEO_CONFERENCE_ELEMENT) {
+			scopeRessource = await this.columnBoardService.findById(scopeId);
```

Comment:
Why is the `scopeResource` a `ColumnBoard`? Shouldn't that be a `VideConferenceElement`? After all the authorization should be perfectly possible on the element. And if we need access to the root `ColumnBoard` that should be possible with something like `element.rootId`.